projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+{
+    "block.doggytalents.dog_bed": "狗窝", 
+    "block.doggytalents.dog_bed.set_owner_help": "空手时，按住潜行键+右键（或手持骨头右键点击）狗窝，最近的站立狗狗会过来并认领此床！若狗狗不幸死亡，右键点击狗窝即可复活！", 
+    "block.doggytalents.dog_bed.owner": "这是%s的床！",
+    "block.doggytalents.dog_bed.reclaim": "%s重新认领了它的床",
+    "block.doggytalents.food_bowl": "食盆", 
+    "block.doggytalents.dog_bath": "狗狗浴缸",
+
+    "item.doggytalents.throw_bone": "投掷骨头",
+    "item.doggytalents.throw_bone_wet": "沾满唾液的投掷骨头",
```

您好，旧物品译名请跟原先的[Doggy Talents](https://www.mcmod.cn/item/list/206-1.html)保持一致，例如此处应该是：
```suggestion
    "item.doggytalents.throw_bone_wet": "沾满口水的投掷骨头",
```

后续物品请您认真比对并修改。